scripts/uma-rewards/upunk-rewards.js

```diff
@@ -1,26 +1,54 @@
 let Web3 = require("web3")
-let web3 = new Web3('http://192.168.1.39:8545')
+let web3 = new Web3('https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161')
 let ABIS = require('./abis.js')
 const WETH = new web3.eth.Contract(ABIS.ERC20_ABI, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2')
 const USDC = new web3.eth.Contract(ABIS.ERC20_ABI, '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48')
 let paymentPeriods = [
   {
-    startBlock: 12680434,
-    amount: 5000000000000000000000n * 82n / 100n,
-    endBlock: 12725501,
+    // 1633046400 | 01.10 | scUMA-1103
+    startBlock: 13330090,
+    amount: 2885000000000000000000n * 90n / 100n,
+    endBlock: 13527858,
   },
+  /*
   {
-    startBlock: 12725501,
-    amount: 5000000000000000000000n * 82n / 100n,
-    endBlock: 12770312,
+    // 1635724800 | 01.11 | scUMA-1202
+    startBlock: 13527858,
+    amount: 6316000000000000000000n * 90n / 100n,
+    endBlock: 13680218,
+  },
+  */
+  /*
+  {
+    // 1635724800 | 25.11 | UMA
+    startBlock: 13680218,
+    amount: 1772000000000000000000n * 90n / 100n,
+    endBlock: 13717846,
+  },
```

Comment:
does that account for anything other than the previously kept rewards on the multisig, so `7346.91822932` uma tokens will be left for the treasury, correct?

Reply:
This is for the rewards that UMA send us to compensate the failed success token deployment for the last week of november.